libraries/controllers/src/controllers/ScriptingInterface.cpp

```
@@ -94,6 +94,25 @@ namespace controller {
         Endpoint::Pointer _second;
     };
 
+    class ActionEndpoint : public Endpoint {
+    public:
+        ActionEndpoint(const UserInputMapper::Input& id = UserInputMapper::Input(-1))
```

should default to INVALID_INPUT